packages/mesh/src/Rope.js

```diff
@@ -35,11 +34,11 @@ export default class Rope extends Mesh
         this.points = points;
         this.refresh();
     }
-
     /**
      * Refreshes
+     *
      */
-    refresh()
+    _refresh()
```

Comment:
Please mark this as @private

Reply:
its supposed to be overriden in generated meshes. There's no "protected" or "default" java-style visibility here, so im sticking with public.